### PR TITLE
Hex

### PR DESCRIPTION
**Pull request by bean710**
This adds %x and %X which print hexadecimal versions of any unsigned integer in lowercase or uppercase, respectively.

### CODE REVIEW

File: printf.c [hunk 1]
```diff
@@ -111,6 +113,10 @@ void *get_mem(spec s, va_list *valist)
 				return (NULL);
 			*(char **)vp = va_arg(*valist, char *);
 			return (vp);
+<<<<<<< HEAD
```
Comment (Beardocracy) [inline printf.c]: Why is this appearing here?

Comment (Beardocracy) [reply printf.c]: And lines 117-119

File: holberton.h [hunk 2]
```diff
@@ -25,9 +25,14 @@ void print_number(int n);
 int _pow(int a, int b);
 int getnum(int num, int index);
 int getlen(int num);
+int base_converter(unsigned int, unsigned int, char *);
+
```
Comment (Beardocracy) [inline holberton.h:29]: Do we need the spaces?

File: holberton.h [hunk 3]
```diff
@@ -25,9 +25,14 @@ void print_number(int n);
...
 int print_decimal(void *);
 int print_string(void *);
 int print_char(void *);
+int print_hex(void *);
+int print_hex_u(void *);
+
```
Comment (Beardocracy) [inline holberton.h:35]: Space?

**Review by Beardocracy — COMMENTED** [review]
Please review my below comments.

File: hexadecimal_print.c [hunk 4]
```diff
@@ -0,0 +1,55 @@
+#include "holberton.h"
+#include <stdio.h>
```
Comment (bean710) [inline hexadecimal_print.c]: Get rid of this line

**Pull request merged**